contribution_to_project.rst

```diff
@@ -0,0 +1,102 @@
+***********************
+Contributing to Ansible
+***********************
+
+This document describes many different ways you can contribute to the Open Source projects of the Ansible ecosystem.
+
+.. contents::
+   :local:
+
+Finding the right contribution type for you
+-------------------------------------------
+
+A contribution to an Open Source project is any positive addition, change, or feedback given by an individual or company to the project.
+
+There are some ideas on how you can contribute:
+
+- :ref:`Contributing to the codebase<Contributing to codebase>`.
+- :ref:`Contributing to the code developer experience<Contributing to dev exp>`.
+- :ref:`Contributing to the user experience<Contributing to user exp>`.
+- :ref:`Contributing to the community<Contributing to community>`.
+- :ref:`Other kinds of contributions<Other kinds of contributions>`.
+
+Most types of contribution do not require programming skills.
+
+.. _Contributing to codebase:
+
+Contributing to the codebase
+----------------------------
+
+Contribution to software itself (its codebase) includes:
+
+- `Proposing new features <https://docs.ansible.com/ansible/latest/community/reporting_bugs_and_features.html#requesting-a-feature>`_.
+- `Reporting bugs <https://docs.ansible.com/ansible/latest/community/reporting_bugs_and_features.html#reporting-a-bug>`_.
+- Participating in design and implementation of new components in dedicated issues, pull requests, GitHub discussions, and `IRC channels <https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels>`_.
+- Fixing bugs.
+- Reviewing code.
+- Refactoring code.
+- Improving stability of an existing codebase through improving a `unit <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_testing.html#adding-unit-tests>`_ and `integration <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_testing.html#adding-integration-tests>`_ testing codebase and infrastructure.
+- Maintaining repositories (for example, backporting merged pull requests and releasing).
+
+You can contribute to a codebase even if you do not have programming skills. For example, you can review proposals, test pull requests locally, add integration tests, file bug reports, create feature requests, or provide feedback to programmers in issues and pull requests.
+
+See also:
+
+- `Contributing to a codebase guide <contributing.rst>`_ to learn how to contribute to collection codebases.
+- `Create a pull request quick-start guide <create_pr_quick_start_guide.rst>`_.
+- `Review checklist <review_checklist.rst>_`.
+- `Test a pull request locally guide <test_pr_locally_guide.rst>`_.
+- `Collection maintainer guidelines <maintaining.rst>`_.
+- `Collection releasing guidelines <releasing_collections.rst>`_.
+
```

Comment:
I don't know enough about github rst to know if these links will work, bu the 'Review Checklist' item looks different from the rest so you may want to review that one a bit.

Reply:
yeah, thanks! Fixed